tutorials/Python/README.md

```diff
@@ -6,15 +6,20 @@ The Oxford Computational Biochemistry Python Tutorial is a series of notebooks w
 
 The tutorial is broken down into 12 sections which progressively introduce various python concepts. We recommend that these be followed in order, however those already acquainted with python may wish to skip ahead to some of the later topics.
 
-## Requirements
+## Getting started
 
-Python 3.6 or higher is required for this workshop. Note that Python 2.x will reach End of Life (EOL) the 1st of January 2020. We advise that no new python code be written in Python 2.x.
+Please see the [setup](setup.md) instructions to get started with the tutorial (installation, etc...).
+Note: the instructions included here are tailored for attendees of the Oxford Computational Biochemistry workshop.
 
 ### Conda
 
 This tutorial uses several [libraries](#Libraries) which are not provided in standard python installations. Whilst there are many means by which to install and manage python libraries, we recommend using the `conda` manager which is provided in [Anaconda python distributions](https://www.anaconda.com/distribution/).
```

Comment:
Better to suggest miniconda?

Reply:
Can do, I'll move the workshop stuff up though to make it clear that they should just install anaconda as per the main setup instructions.